src/model/control_desk.py

```diff
@@ -250,27 +267,15 @@ def color(self, color: ColorHSI) -> None:
 
 
 class FaderBank:
-    """Store a bank page of columns the user can switch through.
-
-    Warning: As of the time of this writing, the buttons for scrolling through a bank are not implemented in fish,
-     hence the user can only access up to 8 columns.
-    """
+    """Store a bank page of columns the user can switch through."""
```

Comment:
The warning should persist, as this is still the case. we can switch between banks but cannot scroll within one. Attaching another xtouch (extender) will also not yet provide further space for a bank.